python/nano/src/bigdl/nano/pytorch/lightning.py

```diff
@@ -21,23 +21,27 @@
 from torch import nn, Tensor
 from torch.nn.modules.loss import _Loss
 from torch.optim import Optimizer
+from torch.optim.lr_scheduler import _LRScheduler
 
 
 class LightningModuleFromTorch(LightningModule):
     def __init__(self, model: nn.Module, loss: _Loss = None, optimizer: Optimizer = None,
+                 scheduler: _LRScheduler = None,
```

Comment:
Why this change? This PR should focus on OpenCV support; you may submit a separate PR for this change if needed.